.github/workflows/ci.yml

```diff
@@ -0,0 +1,125 @@
+            os: ubuntu-latest
+            pyversion: '3.6'
+            TEST_UNIT: 1
+          - name: Linux py37 full
+            os: ubuntu-latest
+            pyversion: '3.7'
+            TEST_UNIT: 1
+            TEST_FULL: 1
+          - name: Linux py38
+            os: ubuntu-latest
+            pyversion: '3.8'
+            TEST_UNIT_NO_INTERNET: 1
+          - name: Linux py39
+            os: ubuntu-latest
+            pyversion: '3.9'
+            TEST_UNIT: 1
+          # Special OS's and interpreters
+          - name: Linux pypy3
+            os: ubuntu-latest
+            pyversion: 'pypy3'
+            TEST_UNIT: 1
+          - name: MacOS py38
+            os: macos-latest
+            pyversion: '3.8'
+            TEST_UNIT: 1
+          - name: Windows py38
+            os: windows-latest
+            pyversion: '3.8'
+            TEST_UNIT: 1
+
+    steps:
+
+      - uses: actions/checkout@v2
+
+      - name: Set up Python ${{ matrix.pyversion }}
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.pyversion }}
+
+      - name: Install dependencies for style build
+        if: matrix.TEST_STYLE == 1
+        run: |
+            python -m pip install -q -U pip
+            pip install -q -U invoke
+            pip install -q -U flake8 black
+            pip install -q -U sphinx numpydoc
+            pip install -q -U numpy pillow
+
+      - name: Install dependencies for tests
+        if: matrix.TEST_UNIT == 1 || matrix.TEST_UNIT_NO_INTERNET == 1
+        shell: bash
+        run: |
+            python -m pip install -q -U pip
+            pip install -q -U invoke
+            pip install -q -U pytest pytest-cov psutil
+            # Install imagio-ffmpeg because its optional but we still want to test it
+            pip install -q -U imageio-ffmpeg
+            # The first local install is really to get the dependencies (as specified in setup.py)
+            pip install .
+            # This step is to force the download of standard images
+            python setup.py sdist
+            # Install again (with images)
+            pip install .
+            # Delete the local copy, so we're actually using the installed version
+            rm -rf ./imageio ./build ./egg-info
+
+      - name: Install optional dependencies for tests
+        if: matrix.TEST_FULL == 1
+        run: |
+            pip install -q -U simpleitk astropy
+
+      - name: Lint (black and flake8)
+        if: matrix.TEST_STYLE == 1
+        run: |
+            invoke lint
+            invoke checkformat
+
+      - name: Test docbuilding
+        if: matrix.TEST_STYLE == 1
+        run: |
+            invoke docs --clean --build
+
+      - name: PR size check
+        uses: ookami-kb/gh-pr-size-watcher@v1
+        if: matrix.TEST_STYLE == 1
+        with:
+          githubToken: ${{ secrets.GITHUB_TOKEN }} # required
+          errorSize: 500
+          warningSize: 300
+          excludeTitle: 'PR_SIZE_SKIP'
```

Comment:
I'm not sure I get the idea behind this action; it could be that I misunderstand the step. Why would we want to fail if the PR has more than 500 lines of code?

Reply:
We had something similar in the Travis CI. It's basically a guard for accidentally committing unintended huge files, which will then be in your Git history forever. I'm not even sure if this would work for binary files though, since it counts lines. Maybe we should just ditch it ...

Reply:
> My vote would go to removing this step.

Agreed